src/Sarif.Driver/DriverResources.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.CodeAnalysis.Sarif.Driver {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "15.0.0.0")]
```

Comment:
>15 [](start = 116, length = 2)

Oops, need to update VS. This file will flicker again the next time the code base is touched.